### PR TITLE
Improve homepage load

### PR DESCRIPTION
**Pull request by gshilin**
1. Update modules
2. Remove keycloack initialization in SSR
3. Cache redux selectors
4. Probably something else...

### CODE REVIEW

File: src/helpers/i18nnext.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ i18n
 
     react: {
       wait: true, // globally set to wait for loaded translations in withTranslation hoc
+      useSuspense: true,
```
Comment (bbfsdev) [inline src/helpers/i18nnext.js:48]: Why is that good?
What that does?

File: src/components/Player/Controls/ProgressCtrl.js [hunk 2]
```diff
@@ -7,9 +7,23 @@ import { selectors as player } from '../../../redux/modules/player';
...
+  const [showChild, setShowChild] = useState(false);
+  // Wait until after client-side hydration to show
+  useEffect(() => setShowChild(true), []);
+
+  if (!showChild) {
+    // You can show some kind of placeholder UI here
+    return null;
+  }
+
+  return <Child/>;
```
Comment (bbfsdev) [inline src/components/Player/Controls/ProgressCtrl.js:20]: What this do? Why this is good?

**Review by bbfsdev — APPROVED** [review]

File: src/helpers/i18nnext.js (hunk 1 above)
Comment (gshilin) [reply src/helpers/i18nnext.js:48]: react-i18nnext uses Suspense by default, so this is more declaration to be sure we ARE doing it

File: src/components/Player/Controls/ProgressCtrl.js (hunk 2 above)
Comment (gshilin) [reply src/components/Player/Controls/ProgressCtrl.js:20]: You're welcome to read [a thorough explanation of Dan Abramov](https://gist.github.com/gaearon/e7d97cdf38a2907924ea12e4ebdf3c85)
You're also welcome to ask "Who this Dan Abramov is?" and I'll answer you

Comment (bbfsdev) [reply src/components/Player/Controls/ProgressCtrl.js:20]: Nice peice.

Comment (bbfsdev) [reply src/components/Player/Controls/ProgressCtrl.js:20]: Any interesting information about Dan beside what is obvious?

Comment (gshilin) [reply src/components/Player/Controls/ProgressCtrl.js:20]: No, just wanted to be sure you know the obvious one :)

File: server/renderer.js [hunk 3]
```diff
@@ -91,10 +91,11 @@ function serverRenderSSOAuth(req, res, next, htmlData) {
     </div>
     <script>window.__isAuthApp = true;</script>
     `;
-  const html    = htmlData.replace(/<div id="root"><\/div>/, rootDiv);
+  // TODO GHSILIN const html    = htmlData.replace(/<div id="root"><\/div>/, rootDiv);
```
[the remark this answers is not in the document]
Comment (gshilin) [reply server/renderer.js]: Now removed it

**Pull request merged**